docs/source/spawners.md

```diff
@@ -163,4 +163,6 @@ When `Spawner.start` is called, this dictionary is accessible as `self.user_opti
 [Spawner]: https://github.com/jupyter/jupyterhub/blob/master/jupyterhub/spawner.py
 
 
-## Writing a custom spawner
\ No newline at end of file
+## Writing a custom spawner
+
+If you are interested in building a custom spawner, you can read [this tutorial](http://jupyterhub-tutorial.readthedocs.io/en/latest/spawners.html)
```

Comment:
Hi @willingc , sorry, trying to fix it now. I'm having problems with git because I'm trying to do another commit, push and then `git remote add upstream https://github.com/jupyterhub/jupyterhub.git` and I get the error `remote upstream already exists.`.
Should I just try with a different name other than upstream?